spec/menu_spec.rb

```diff
@@ -2,12 +2,16 @@
 
 RSpec.describe Menu do
 
-  let(:mains) { Menu::MAINS }
-  let(:list) { subject.send(:sides_list) }
-
-  describe '#list' do
+  describe '#menu' do
     it 'prints a full list of available dishes' do
-      expect(subject.show).to include list
+      expect{subject.show}.to output.to_stdout
+    end
+  end
+
+  describe '#available?' do
+    it 'raises error if chosen item is unavailable' do
```

Comment:
Consider rewording as this isn't what the test seems to be testing for - maybe "it 'checks if dish is available' do"